src/utils/draftPlantingSiteUtils.ts

```diff
@@ -82,3 +88,31 @@ export const toDraft = (payload: DraftPlantingSitePayload): DraftPlantingSite =>
 
   return draft;
 };
+
+export const fromDraftToCreate = (site: DraftPlantingSite): CreatePlantingSiteRequestPayload => {
+  const {
+    boundary,
+    description,
+    exclusion,
+    name,
+    organizationId,
+    plantingSeasons,
+    plantingZones,
+    projectId,
+    timeZone,
+  } = site;
+
+  const payload: CreatePlantingSiteRequestPayload = {
+    boundary,
+    description,
+    exclusion,
+    name,
+    organizationId,
+    plantingSeasons,
+    plantingZones,
+    projectId,
+    timeZone,
+  };
+
+  return payload;
+};
```

Comment:
Are there additional fields aside from the ones destructed here? Is that why this doesn't just `return { ...site };`?

Reply:
Mostly wanted to respect the API payload. Should be okay in this instance.